opennms-web-api/src/main/java/org/opennms/web/svclayer/model/LocationMonitorListModel.java

```diff
@@ -162,10 +157,7 @@ public int compare(Entry<String, String> one, Entry<String, String> two) {
             });
             for (Entry<String, String> detail : details) {
                 if (
-                    !detail.getKey().equals(PollerBackEnd.HOST_NAME_KEY) && 
```

Comment:
Wait! What?

Reply:
It's just to get it working. Have to revisit this code anyways.